### PR TITLE
Fix/python samples 53 & 54 

### LINKED ISSUE #2640: Error on python sample 54: got multiple values for keyword argument 'type'
## Describe the bug
I got error on fetching module while trying on this python sample codes
https://github.com/microsoft/BotBuilder-Samples/tree/master/samples/python/54.teams-task-module

## To Reproduce
1. python app.py in python sample code number 54
2. Say something to the Bot
3. Click "Adaptive Card" on the Bot response

## Expected behavior
It shows the Card correctly

## Console Output
 [on_turn_error] unhandled error: __init__() got multiple values for keyword argument 'type'
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/botbuilder/core/bot_adapter.py", line 109, in run_pipeline
    return await self._middleware.receive_activity_with_status(
  File "/usr/local/lib/python3.8/site-packages/botbuilder/core/middleware_set.py", line 69, in receive_activity_with_status
    return await self.receive_activity_internal(context, callback)
  File "/usr/local/lib/python3.8/site-packages/botbuilder/core/middleware_set.py", line 79, in receive_activity_internal
    return await callback(context)
  File "/usr/local/lib/python3.8/site-packages/botbuilder/core/activity_handler.py", line 76, in on_turn
    invoke_response = await self.on_invoke_activity(turn_context)
  File "/usr/local/lib/python3.8/site-packages/botbuilder/core/teams/teams_activity_handler.py", line 133, in on_invoke_activity
    await self.on_teams_task_module_fetch(
  File "/Users/alfianramadhan/Projects/Accellion/botbuilder-samples/samples/python/54.teams-task-module/bots/teams_task_module_bot.py", line 58, in on_teams_task_module_fetch
    continue_response = TaskModuleContinueResponse(type="continue", value=task_info)
  File "/usr/local/lib/python3.8/site-packages/botbuilder/schema/teams/_models_py3.py", line 1618, in __init__
    super(TaskModuleContinueResponse, self).__init__(type="continue", **kwargs)
TypeError: __init__() got multiple values for keyword argument 'type'

## Screenshots
image

## Additional context
In Javascript sample codes it works fine. But in Python sample codes it does not work.

[bug]

### PR DESCRIPTION
**Pull request by Josverl**
Fixes #2640 in Python sample 54. Task Module

the offending line is :  
``` python
        continue_response = TaskModuleContinueResponse(type="continue", value=task_info)
```
a similar issue is with line:
``` python
message_response = TaskModuleMessageResponse(type="message", value="Thanks!")
```

## Proposed Changes

I think that the **kwargs lead to a duplication of the `type` parameter when this is also provided as a parameter to `TaskModuleContinueResponse`, hence the error : **`TypeError: init() got multiple values for keyword argument 'type'`**
[ref: SO](https://stackoverflow.com/questions/48731490/multiple-values-for-keyword-argument-when-using-kwargs)

**corrected sample:**
below is a simple correction to the sample,

``` python 
#line 58
        continue_response = TaskModuleContinueResponse( value=task_info)
#line 70
        message_response = TaskModuleMessageResponse( value="Thanks!")
```
works like a charm.
pybot

## Testing
no new features added

### CODE REVIEW

**Comment by Josverl** [comment]
sorry , need to revert and clean up other files

**Comment by Josverl** [comment]
Added similar fixes for the 53 example.

note that the sample still runs into errors when starting from the CmdBox or from a personal scope. 
I suspect that is a functional issue ( the message flow is broken as is is not possible for a post to be sent from these scopes )

**Comment by jwiley84** [comment]
Looks good to me so far. Let me look into your comment about scopes and I should get this approved today, hopefully

**Review by jwiley84 — APPROVED** [review]
Looks good to  me

**Review by axelsrz — APPROVED** [review]

File: samples/python/54.teams-task-module/bots/teams_task_module_bot.py [hunk 1]
```diff
@@ -51,11 +51,11 @@ async def on_teams_task_module_fetch(
                 "actions": [{"type": "Action.Submit", "title": "Submit",}],
             }
         )
-
+        
         task_info = TaskModuleTaskInfo(
             card=card, title="Adaptive Card: Inputs", height=200, width=400
         )
-        continue_response = TaskModuleContinueResponse(type="continue", value=task_info)
+        continue_response = TaskModuleContinueResponse( value=task_info)
```
Comment (Virtual-Josh) [inline samples/python/54.teams-task-module/bots/teams_task_module_bot.py:58]: Could you remove the white space? `...Response( value...)`

File: samples/python/53.teams-messaging-extensions-action-preview/bots/messaging_extension_action_preview_bot.py [hunk 2]
```diff
@@ -47,7 +47,7 @@ async def on_teams_messaging_extension_fetch_task(
         task_info = TaskModuleTaskInfo(
             card=card, height=450, title="Task Module Fetch Example", width=500
         )
-        continue_response = TaskModuleContinueResponse(type="continue", value=task_info)
+        continue_response = TaskModuleContinueResponse( value=task_info)
```
Comment (Virtual-Josh) [inline samples/python/53.teams-messaging-extensions-action-preview/bots/messaging_extension_action_preview_bot.py:50]: Could you remove the white space here? `...Response( value...)`

File: samples/python/54.teams-task-module/bots/teams_task_module_bot.py [hunk 3]
```diff
@@ -67,7 +67,7 @@ async def on_teams_task_module_submit(
             )
         )
 
-        message_response = TaskModuleMessageResponse(type="message", value="Thanks!")
+        message_response = TaskModuleMessageResponse( value="Thanks!")
```
Comment (Virtual-Josh) [inline samples/python/54.teams-task-module/bots/teams_task_module_bot.py:70]: Same thing here

**Review by Virtual-Josh — APPROVED** [review]
A couple of nits, but LGTM.

**Review by axelsrz — APPROVED** [review]

**Pull request merged**